backend/api/src/main/kotlin/io/tolgee/api/v2/controllers/notification/NotificationController.kt

```diff
@@ -54,16 +57,35 @@ class NotificationController(
     pageable: Pageable,
     @ParameterObject
     filters: NotificationFilters = NotificationFilters(),
-  ): PagedModel<NotificationModel> {
+    cursor: String? = null,
+  ): PagedModelWithNextCursor<NotificationModel> {
+    if (cursor != null &&
+      pageable.pageNumber > 0 &&
+      pageable.sort.toString() != "id: DESC"
```

Comment:
I would not rely on the toString method, rather I check that the sort is default sort.

Also! I recommend to use `createdAt: desc, id: desc` for sorting. Because in some cases the date can be higher when id can be lower, because the id is allocated before the actual insert so there can be a race condition.